services/keyboard/src/org/chromium/mojo/keyboard/InputConnectionAdaptor.java

```diff
@@ -71,4 +83,20 @@ public boolean setSelection(int start, int end) {
         mClient.setSelection(start, end);
         return super.setSelection(start, end);
     }
+
+    // Number keys come through as key events instead of commitText!?
```

Comment:
This was preexisting code that had originally been checked into the Sky repository, was synced over to the Mojo side, and now is coming back to Sky through this roll.

I'd prefer to keep it as is throughout the sync process.